apps/studio/src/features/editing-experience/components/form-builder/FormBuilder.tsx

```diff
@@ -45,7 +45,8 @@ const renderers: JsonFormsRendererRegistryEntry[] = [
   },
   { tester: jsonFormsIntegerControlTester, renderer: JsonFormsIntegerControl },
   { tester: jsonFormsTextControlTester, renderer: JsonFormsTextControl },
-  { tester: jsonFormsOneOfControlTester, renderer: JsonFormsOneOfControl },
+  { tester: jsonFormsAllOfControlTester, renderer: JsonFormsAllOfControl },
+  { tester: jsonFormsAnyOfControlTester, renderer: JsonFormsAnyOfControl },
```

Comment:
this is definitely a me thing, i don't quite get how `oneOfTester` translates over. is it just `allOf` a single thing?

Reply:
`oneOf` is very similar to `anyOf` in the JSON Schema context. `oneOf` means that exactly one of the schemas in the array must fit, whereas `anyOf` means at least one.

In our case, we use `anyOf` because that is the one supported by TypeBox (they don't support `oneOf`, see PR description). We are also safe because we use a constant for differentiating between items in the `anyOf` schema (e.g. `variant` in the Hero component).

`allOf` is slightly different, because it is the union of all the properties in its list of schemas.